### ISSUE TITLE
Duplicate symbol FBSDKVideoUploader in FBSDKShareKit and FBSDKGamingServicesKit

### ISSUE THREAD

**Issue by qbit86** [facebook/facebook-ios-sdk#1512]
## Checklist

- [x] I've updated to the latest released version of the SDK: [8.0.0](https://github.com/facebook/facebook-ios-sdk/releases/tag/v8.0.0)
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md) — **the link in the issue template is broken**, should be: https://github.com/facebook/facebook-ios-sdk/blob/master/CODE_OF_CONDUCT.md
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

> Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Unity 2018.4.10f1
- Xcode Version: `11.5`
- Installation Platform & Verison: `Manual`, [facebook-unity-sdk-8.0.0.zip](https://developers.facebook.com/docs/unity/), [Facebook iOS SDK 8.0.0](https://github.com/facebook/facebook-ios-sdk/releases/tag/v8.0.0), [FBAudienceNetwork-6.0.0.zip](https://developers.facebook.com/docs/audience-network/guides/adding-sdk/ios)

## Goals

> What do you want to achieve?

For mobile cross-platform Unity game I want to update Facebook SDK and make it successfully build on our buildbot.

## Expected Results

> What do you expect to happen?

After updating to the latest version of the Facebook SDK I expect builds to succeed.

## Actual Results

> What actually happened? Can you provide a stack trace?

Buildbot reports a linking failure on the step “Xcode archive” with the following messages:
```
duplicate symbol '_OBJC_IVAR_$_FBSDKVideoUploader._parameters' in:
    /Users/buildserver/.../FBSDKShareKit.framework/FBSDKShareKit(FBSDKVideoUploader.o)
    /Users/buildserver/.../FBSDKGamingServicesKit.framework/FBSDKGamingServicesKit(FBSDKVideoUploader.o)
...
duplicate symbol '_OBJC_IVAR_$_FBSDKVideoUploader._uploadSessionID' in:
    /Users/buildserver/.../FBSDKShareKit.framework/FBSDKShareKit(FBSDKVideoUploader.o)
    /Users/buildserver/.../FBSDKGamingServicesKit.framework/FBSDKGamingServicesKit(FBSDKVideoUploader.o)
ld: 12 duplicate symbols for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** ARCHIVE FAILED **
```

## Steps to Reproduce

> What are the steps necessary to reproduce this issue?

I don't know how to reproduce it in a clean Xcode project. But in case of Unity it's probably sufficient just to drop these folders from [Facebook SDK v8.0.0](https://github.com/facebook/facebook-ios-sdk/releases/tag/v8.0.0) to the project:
* FBSDKCoreKit.framework
* FBSDKGamingServicesKit.framework
* FBSDKLoginKit.framework
* FBSDKShareKit.framework

## Code Samples & Details

> Please provide a code sample, as well as any additional details, to help us track down the issue.

It's probably related to this pull request: [Moving video uploader into GamingServicesKit #1494](https://github.com/facebook/facebook-ios-sdk/pull/1494)  
`FBSDKVideoUploader` was moved from FBSDKShareKit to FBSDKGamingServicesKit, but the older version of FBSDKShareKit was released, or something like that.

**Comment by IArapov** [comment]
Reproduced the same issue with Unity2019 and xCode 12.0.0

**Comment by qbit86** [comment]
Are there any estimates on when the fixed release will be available?

@jingping2015 @joesus @kalendae @robtimp

**Comment by firatagdas** [comment]
Same issue with Unity 2018.4.23f1 and Xcode 12. I had to strip symbols from FBSDKShareKit

**Comment by joesus** [comment]
This should be fixed in next release - commit - 0cd91819be1a2c93aa04dc569a5623caf4636975
